cmds/msr/msr_linux.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2012-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// This file contains support functions for io for Linux.
+package main
+
+import (
+	"encoding/binary"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+func msrList(n string) []string {
```

Comment:
I actually just removed the check and went with a log.Fatalf in msrList. If you don't get anything you're not going anywhere.